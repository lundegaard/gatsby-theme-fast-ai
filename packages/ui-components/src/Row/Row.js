import React from 'react';

import Flex from '../Flex';

const Row = props => <Flex mx={-2} {...props} />;

export default Row;
