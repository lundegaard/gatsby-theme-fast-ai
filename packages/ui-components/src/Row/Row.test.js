import React from 'react';
import { shallow } from 'enzyme';

import Row from './Row';

describe('<Row />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<Row {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
