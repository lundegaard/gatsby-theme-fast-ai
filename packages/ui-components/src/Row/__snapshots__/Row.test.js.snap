// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> matches snapshot when shallowly rendered 1`] = `
<Flex
  mx={-2}
/>
`;
